CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/ActivityExtensions.cs

```diff
@@ -142,6 +147,31 @@ public static class ActivityExtensions
             return activity;
         }
 
+        /// <summary>
+        /// Notifies the mentioned users.
+        /// </summary>
+        /// <param name="replyActivity">The reply activity.</param>
+        /// <returns>Modified activity.</returns>
+        public static Activity NotifyMentionedUsers(this Activity replyActivity)
```

Comment:
Can we make these extension methods on the IMessageActivity interface instead of the Activity class? The BotBuilder Dialogs framework tends to return interfaces, so this would avoid casting. (Also, the BotBuilder SDK defines extension methods on interfaces.)